src/devices/Common/Iot/Device/Graphics/MouseClickSimulatorUInput.uinput.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this feels like it should be in separate PR and is a bit out of scope for this repo